src/commands/notify.yml

```diff
@@ -129,6 +129,8 @@ steps:
     steps:
     - run:
         command: |
-          curl -X POST -H"Content-Type:application/json" -H "Authorization: GenieKey ${OPS_GENIE_API_KEY}" -d @/tmp/webhook.json https://api.opsgenie.com/v2/alerts
+          cat /tmp/raw-webhook.json | jq '.details.outcome="failed"'  > /tmp/webhook.json
+          mv /tmp/webhook_temp.json /tmp/raw-webhook.json
```

Comment:
What's the `webhook_temp.json` file for?

Reply:
You cannot do ```cat /tmp/raw-webhook.json | jq '.details.outcome="failed"'  >  /tmp/raw-webhook.json```, it overwrites the file, need intermediate file to save the result.